packages/docs/content/foundation/typography/body.mdx

```diff
@@ -0,0 +1,153 @@
+
+The default font size for most content within HealthCare.gov.
+
+</ThemeContent>
+
+<EmbeddedExample>
+  <p class="ds-text-body--md">{loremIpsumGenerator('l')}</p>
+</EmbeddedExample>
+
+### Small body text
+
+`.ds-text-body--sm` 
+
+<ThemeContent onlyThemes={['healthcare']}>
+
+Only used for footnotes, fine print, and legal jargon. 
+
+</ThemeContent>
+
+<EmbeddedExample>
+  <p class="ds-text-body--sm">{loremIpsumGenerator('l')}</p>
+</EmbeddedExample>
+
+## Code 
+
+
+### Responsive typography
+
+The `.ds-type-heading-3xl`, `ds-type-heading-2xl`, and `ds-type-heading-xl` classes are responsive by default.
+To apply responsive typography elsewhere, use the [font size utility class](/utilities/font/font-size/#responsive-font-sizes).
+Since the base typography margins and line height is measured in `em` units,
+they'll automatically adjust as you change the font size.
+
+<ResponsiveExample
+  storyId="typography-responsive--headings"
+  title="Responsive typography example"
+/>
+
+### Styles
+
+The following Sass variables can be overridden to change the type sizes:
+
+- `$font-size-5xl`
+- `$font-size-4xl`
+- `$font-size-3xl`
+- `$font-size-2xl`
+- `$font-size-xl`
+- `$font-size-lg`
+- `$font-size-md`, uses `$base-font-size`
+- `$font-size-sm`
+
+## Guidance
+
+<ThemeContent onlyThemes={['healthcare']}>
+
+### Don't 
+- Don't use italic text.
+
+### Mobile and small screens
+Header sizes should be reduced on mobile screen sizes. Header sizes can be reduced depending on the designers discretion but can generally use this guide for reducing your header sizes.
+
+#### Body
+Small to Large Body text - No change in size unless header size is smaller than the body size
+
+### Font Weights
+There is only one bold font option for paragraph styles (Open Sans font). Open Sans 600 (Semibold) is the only bold option that should be used. There are no bold options for headers.
+
+
+### Layout
+
+There should always be a parent/child relationship between headers and content on the page. Every header should have a relationship to something else on the page. This can apply to many different relationships like a header and a form group or a header and list of links. Stray headers or content should be reevaluated.
+
+- Paragraph styles should never be used as a parent to another text element.
+- Header elements should never have equal or smaller font size than the paragraph element.
+- Not all paragraphs need a header, but all headers need children.
+- Every page should have one heading 1 (H1)
+- Headings style appearing under H1 should follow a logical order for information design and supporting various assistive technology devices/software.
+
+### Captions 
+
+Generally, most content on Healthcare.gov should be 16px or higher. 
+Captions, disclaimers, footnotes and text that is there for legal reasons can use the small heading and body styles.
+
+### Phone Numbers
+- All site provided phone numbers will be shown as links (future update).
```

Comment:
Should we have a PhoneLink component added to our backlog to make sure this update happens? Brief search in our backlog didn't return any component work from what I could tell.

Reply:
Couldn't hurt!